e2e/wdio/core/tests/tabs.e2e-spec.ts

```diff
@@ -0,0 +1,154 @@
+        counterField, contentField, icon1, titleAndCountSection, contentSection, collapsibleTab, acceleratedIcon,
+        fdIcon, filterMode
+    } = tabsPage;
+
+    beforeAll(() => {
+        tabsPage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+        waitForPresent(fdTab);
+    }, 1);
+
+    it('check tabs select', () => {
+        expect(checkTabsSelect(tabsExample)).toBe(true);
+        expect(checkTabsSelect(IconOnlyExample)).toBe(true);
+        expect(checkTabsSelect(ProcessExample)).toBe(true);
+        expect(checkTabsSelect(FilterExample)).toBe(true);
+        expect(checkTabsSelect(AddExample)).toBe(true);
+        expect(checkTabsSelect(collapsibleOverflowExample)).toBe(true);
+        expect(checkTabsSelect(collapsibleExample)).toBe(true);
+        expect(checkTabsSelect(SelectionExample)).toBe(true);
+        expect(checkTabsSelect(stackendContentExample)).toBe(true);
+    });
+
+    it('add-remove tab testing', () => {
+        const lengthBefore = getElementArrayLength(AddExample + fdTab);
+        click(addBtn);
+        let lengthAfter = getElementArrayLength(AddExample + fdTab);
+        expect(lengthAfter).toBeGreaterThan(lengthBefore);
+        doubleClick(removeBtn);
+        lengthAfter = getElementArrayLength(AddExample + fdTab);
+        expect(lengthBefore).toBeGreaterThan(lengthAfter)
+        for (lengthAfter; lengthAfter != 1; lengthAfter--)
+            click(removeBtn)
+        click(removeBtn)
+        lengthAfter = getElementArrayLength(AddExample + fdTab);
+        // Check that the last tab can not be removed
+        expect(lengthAfter).toEqual(1)
+
+    });
+
+    it('Check choosing tabs via buttons', () => {
+        click(chooseTabsBtn, 1)
+        expect(getAttributeByName(SelectionExample + fdTab, 'aria-selected', 1)).toEqual('true')
+        click(chooseTabsBtn, 0)
+        expect(getAttributeByName(SelectionExample + fdTab, 'aria-selected', 0)).toEqual('true')
+        expect(getAttributeByName(SelectionExample + fdTab, 'aria-selected', 1)).toEqual('false')
+    });
+
+    it('check collapsible overflow', () => {
+        const length = getElementArrayLength(collapsibleOverflowExample + collapsibleTab);
+        const lastPointOfMainList = getText(collapsibleOverflowExample + collapsibleTab, length - 1);
+        click(moreBtn)
+        const firstPointOfExpandedList = getText(expandedListPoint, 0);
+        click(expandedListPoint, 0);
+        expect(getText(expandedListPoint, 0)).toEqual(lastPointOfMainList);
+        expect(getText(collapsibleOverflowExample + collapsibleTab, length - 1)).toEqual(firstPointOfExpandedList);
+    });
+    it('Should check collapsible tabs', () => {
+        click(collapsibleExample + fdTab, 2);
+        expect(getAttributeByName(tabPanel, 'aria-expanded', 2)).toEqual('true');
+    });
+
+
+    describe('Tabs constructor testing', () => {
+        it('filter mode', () => {
```

Comment:
needs better description.
better is: it('should check filter mode
or
'should check the ability to enable filter mode' 
etc